docs/community/20230104-mixed-clusters.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Feature Group for Mixed Clusters in Cluster API
+authors:
+  - "@prometherion"
+  - "@lentzi90"
+reviewers:
+  - "@lentzi90"
+creation-date: 2023-01-04
+last-updated: 2023-01-04
+status: proposed
+see-also:
+  - https://github.com/kubernetes-sigs/cluster-api/issues/7475#issuecomment-1302454745
+  - https://kubernetes.slack.com/archives/C8TSNPY4T/p1667458956589639
+---
+# Mixed Clusters in Cluster API Feature Group
+
+This document briefly outlines the scope, communication media, and stakeholders for a formal Feature Group dedicated to defining a Cluster API-approved solution for "mixed clusters" which combines two or more infrastructure providers.
+
+## User Story and Problem Statement
+
+As a Kubernetes cluster administrator, I want the ability to specify different infrastructure providers for the Control Plane and the Worker nodes to decouple each-other infrastructure providers.
+
+At the current state, Cluster API is giving for granted that worker nodes and control plane components are backed by the same infrastructure provider. However, this is a hard statement, since the two components could be backed by different infrastructure providers.
+
+A practical example is offered with [Kamaji](https://github.com/clastix/kamaji) that offers virtualized Control Plane resources used by worker nodes provisioned on any cloud provider by using the `kubeadm` bootstrapper.
+
+## Scope
+
+The scope of this group will be the following:
+
+1. Creating a PoC demonstrating that a mixed cluster solution is possible
+2. Design the new API resources allowing to specify of different Infrastructure providers
+3. Implementing the new API to support this feature request
+
+## Communication
+
+We will bi-weekly meet on [Monday at 7:00 PM (CET)][https://zoom.us/j/861487554]. [Convert to your timezone](https://dateful.com/time-zone-converter?t=19:00&tz=CET%20%28Central%20European%20Time%29). Meeting notes will be documented in this [Google Doc](https://docs.google.com/document/d/12TGubbGJa3w4ux8kJnR7sb4l39iHxcOAtbJqLwSlnkc/edit?usp=sharing).
```

Comment:
Due to the initial effort required for this, I'm proposing a bi-weekly meeting on Monday.

Does 7 PM CET work for you, @richardcase @lentzi90?

Reply:
@prometherion - that works for me 👍

Reply:
It does not work for me unfortunately, but I'm not sure how much I can contribute anyway so please don't see it as a blocker. I'm generally available between 7 am and 15 pm CET. Can do later meetings sometimes but not on Mondays.